tests/benchdnn/utils/res.hpp

```diff
@@ -36,6 +36,7 @@ enum res_state_t {
     LISTED,
     INITIALIZED,
     EXECUTED,
+    DEFERRED,
```

Comment:
Let's add a comment explaining this status here since it's not quite conventional judging by its name.